templates/emails/registration-email.tpl

```diff
@@ -7,9 +7,14 @@
     <a href="{{URL}}">Click here to finish registration.</a>
 </p>
 <p>
-Please make sure to update your user profile and set notifications on your profile
+Please make sure to update your profile and set notifications on your profile
 if you want to be notified of new translation projects by email.
 For more information, please view
 <a href="https://community.translatorswb.org/t/signing-up-for-kato-platform-email-notifications/121">
 Signing up for Kató Platform email notifications</a>.
 </p>
+<p>
+If you are a partner organization, please discard the above information as it does not apply to you. 
```

Comment:
Just pointing out something I noticed, in this sentence I believe the word you are looking for is "disregard", not "discard". :)

Reply:
I will make that wording change after I merge the pull request... thanks